conversion/src/main/java/com/qualinsight/mojo/cobertura/transformation/JaxbCoberturaToSonarQubeReportConverter.java

```diff
@@ -75,6 +78,12 @@ public void convertReport(
       final String[] sourcesPaths,
       final String projectPath) throws MojoExecutionException {
     try {
+      //There is no real lookup but the project doesn't have Tuple or KeyValuePair
```

Comment:
Small smell here, using in a Map in not a very map like way, e.g. I only iterate over the EntrySet and don't do any get('key').